docs/source/initialsetup/mameintro.rst

```diff
@@ -46,19 +74,38 @@ The official MAME binaries are compiled and designed to run on a standard Window
 * A Direct3D, or OpenGL capable graphics card
 * Any DirectSound capable sound card/onboard audio
 
-Of course, the minimum requirements are just that: minimal. You may not get optimal performance from such a system, but MAME should run. Modern versions of MAME require more power than older versions, so if you have a less-capable PC, you may find that using an older version of MAME may get you better performance, at the cost of greatly lowered accuracy and fewer supported systems.
+Of course, the minimum requirements are just that: minimal. You may not get
+optimal performance from such a system, but MAME should run. Modern versions of
+MAME require more power than older versions, so if you have a less-capable PC,
+you may find that using an older version of MAME may get you better
+performance, at the cost of greatly lowered accuracy and fewer supported
+systems.
 
-MAME will take advantage of 3D hardware for compositing artwork and scaling the games to full screen. To make use of this, you should have a modern Direct3D 8-capable video card with at least 16MB of video RAM.
+MAME will take advantage of 3D hardware for compositing artwork and scaling the
+games to full screen. To make use of this, you should have a modern Direct3D
+8-capable video card with at least 16MB of video RAM.
```

Comment:
This is kind of complex and not entirely true.  We don’t support Direct3D 8 any more as far as I know, and it’s got to a point where some newer video cards don’t natively support Direct3D 9.  For example Intel UHD Graphics 630 (integrated in Coffee Lake CPUs) lacks Direct3D 9 support.  However, if you install the DirectX 9 runtime, Windows 10 will emulate Direct3D 9 on the CPU.  The performance isn’t as bad as you might expect, but definitely chokes on demanding games.

Trying to explain what kind of video cards work well with which video backends will require a bit more thought.

Reply:
Well, BGFX backend should be fine with any modern Windows machine, even laptops running Intel onboard. We definitely need to change this text.

Reply:
Yeah, BGFX with the d3d11 backend is fine with pretty much any modern GPU.  BGFX with opengl or the OpenGL module work with most video cards provided the drivers aren’t completely broken.  The OpenGL module supports older Intel GPUs than any of the other options IIRC.

Reply:
"most recent computers will work with OpenGL, D3D11, or the BGFX video system"?

Reply:
Think about the phrasing a bit more.  We support opengl, d3d11 and vulkan via bgfx (and I think d3d12 might be working with bgfx as well now), but we also have an opengl module separate from bgfx.

Reply:
"most semi-recent computers will work with the BGFX video system in OpenGL, Direct3D 11, and in many cases Vulkan. We also support non-BGFX basic OpenGL and Direct3D for machines where BGFX doesn't work well."